docs/tutorials/lattice-fermions.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>Might be worth mentioning that if the user has scarce mathematical knowledge he can check if his ansatz is holomorphic, at least on a set of points, by running</p><p> </p><pre class="ql-syntax" spellcheck="false">nk.utils.is_probably_holomorphic(vstate._apply_fun, vstate.parameters, vstate.samples)
</pre><p> </p><p>which in this case will return False.</p><p> </p><p>By the way, for real parameters all the holomorphic logic is completely bypassed, and one could also avoid to specify it altogether..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
good point, just removed holomorphic alltogether